dell/xps/15-7590/README.wiki

```diff
@@ -0,0 +1,49 @@
+= Dell XPS 15 7590 =
+*Mostly copied from 15-9550
+
+== Tested Hardware ==
+
+* CPU: Intel(R) Core(TM) i9-9980HK
+* RAM: 32 GB
+* HDD: 1 TiB SSD
+* Screen: 15" 4k (3840✕2160)
+* Input: Touchscreen and trackpad.
+
+== Firmware Configuration ==
+
+Not much tweaking of NixOS itself was needed. But we currently cannot automate the firmware setup, so this must be done by hand.
```

Comment:
Is it not supported by https://fwupd.org/ ?

Reply:
Not familiar..

Reply:
fwupd is used to update firmware. It can't do anything like the below with custom configurations.